### DISCUSSION TITLE
[Bug] Hosted Checkout styling-settings not working

### DISCUSSION THREAD

**Discussion by marnauortega** [swellstores/community#342] [Help] [answered]
I'm trying to change the background-color of the content area, but it doesn't work. I imagine it's overwritten by another style.
The same happens with the color of the links, I change it but it's overwritten by another blue style.
I hope someone can check if I'm doing something wrong.

On a different note, I want to ask if it would be possible to add a color swatch for the cart area, to pick the font color, and allow to get a white text over a black background, like this:

![Captura de Pantalla 2023-08-15 a les 19 24 55](https://github.com/swellstores/community/assets/37710726/412081a7-057b-4519-988e-086febe55522)

Thanks a lot!

**Comment by logeshswell** [comment]
@marnauortega I'll take a look at this issue and get back to you when an update becomes available.

**Reply by logeshswell** [reply, marked as answer]
This issue is fixed now. Can you check and confirm?

**Reply by marnauortega** [reply]
That is awesome, thank you so much @logeshswell !

Just one thing, do you think it would be possible to have a color picker for the text on the cart area?
I ask because now if I change the cart area color background to black, I can't make the text white.

![Captura de Pantalla 2023-09-14 a les 18 50 42](https://github.com/swellstores/community/assets/37710726/e5858557-2685-4f13-bf2f-78e07681d922)

Thanks again!

**Reply by logeshswell** [reply]
No plans as of now, but you can do it via **Checkout scripts** under Storefront > Checkout settings. For example,
```html
<script>
  document.getElementById("root").style.color = "white";
</script>
```

**Reply by marnauortega** [reply]
Oh, indeed. That's all I neeed. Golden!
Thank you so much @logeshswell
